javascript/node/selenium-webdriver/firefox/index.js

```diff
@@ -367,14 +373,9 @@ class Driver extends webdriver.WebDriver {
       let service = createGeckoDriverService(binary);
       serverUrl = service.start();
       onQuit = () => service.kill();
+      caps.set(Capability.PROFILE, profile.encode());
```

Comment:
This seems unnecessarily expensive for local runs (especially if the profile includes an extension). Could we write the profile to disk and specify the path via capabilities?